src/Device/Driver/ATR833.cpp

```diff
@@ -110,17 +312,44 @@ ATR833Device::PutStandbyFrequency(RadioFrequency frequency,
   return true;
 }
 
+void
+ATR833Device::LinkTimeout()
+{
+  NullOperationEnvironment env;
+
+  ATRBuffer buffer(ALIVE); 
+  buffer.Send(port, env);
+}
+
+void
+ATR833Device::OnSysTicker()
+{
+  // ALIVE shall be send every 5 seconds
+  if (status_clock.CheckUpdate(std::chrono::seconds(5))) {
+    NullOperationEnvironment env;
+
+    ATRBuffer buffer(ALIVE);
+    buffer.Send(port, env);
+
+    ATRBuffer buffer2(REQUESTDATA);
+    buffer2.Send(port, env);
+  }
+}
+
 static Device *
 ATR833CreateOnPort([[maybe_unused]] const DeviceConfig &config, Port &com_port)
 {
-  return new ATR833Device(com_port);
-}
+  Device *dev = new ATR833Device(com_port);
+
+  // call to initiate communication, radio activates protocol afterwards
+  dev->LinkTimeout();
```

Comment:
Ok. Found a better place for this, even though the function name (`EnableNMEA()`) confused me at first because this can be used even without the device itself is using NMEA.